### PR TITLE
feat: add ability to delete an expense

### PR DESCRIPTION
**Pull request by djaiss**
I didn't implement the ability to delete an expense 😨

### CODE REVIEW

**Comment by djaiss** [comment]
@asbiin please review.

File: resources/js/Shared/ExpenseStatus.vue [hunk 1]
```diff
@@ -225,6 +227,10 @@
         </li>
       </ul>
     </div>
+
+    <div v-if="canDelete" class="actions pa3 tc f6">
+      <a class="c-delete pointer" @click.prevent="destroy()">Delete the expense</a>
```
Comment (asbiin) [inline resources/js/Shared/ExpenseStatus.vue:232]: message needs to be localized

File: routes/web.php [hunk 2]
```diff
@@ -170,9 +170,9 @@
                     Route::get('', 'Company\\Employee\\Administration\\EmployeeAdministrationController@show')->name('employees.administration.show');
 
                     // expenses
-                    Route::resource('expenses', 'Company\\Employee\\Administration\\Expenses\\EmployeeExpenseController', ['as' => 'employee.administration'])->only([
-                        'index', 'show',
-                    ]);
+                    Route::get('expenses', 'Company\\Employee\\Administration\\Expenses\\EmployeeExpenseController@index')->name('employee.administration.expenses.index');
+                    Route::get('expenses/{expense}', 'Company\\Employee\\Administration\\Expenses\\EmployeeExpenseController@show')->name('employee.administration.expenses.show');
+                    Route::delete('expenses/{expense}', 'Company\\Employee\\Administration\\Expenses\\EmployeeExpenseController@destroy');
```
Comment (asbiin) [inline routes/web.php:175]: you could add a `name` and use it in the vue with `this.route(...)`

**Review by asbiin — APPROVED** [review]
Some remarks, but it's ok on the overall 👍🏽

**Pull request merged**